osquery/events/linux/bpf/bpfeventpublisher.cpp

```diff
@@ -152,12 +218,45 @@ Status BPFEventPublisher::setUp() {
 
     auto& buffer_storage = *buffer_storage_it->second.get();
 
-    auto function_tracer_exp =
-        tob::ebpfpub::IFunctionTracer::createFromSyscallTracepoint(
-            tracer_allocator.syscall_name,
-            buffer_storage,
-            *d->perf_event_array.get(),
-            kEventMapSize);
+    tob::StringErrorOr<ebpfpub::IFunctionTracer::Ref> function_tracer_exp;
+
+    if (tracer_allocator.kprobe) {
+      if (tracer_allocator.parameter_list == nullptr) {
+        LOG(ERROR) << "The kprobe for " << tracer_allocator.syscall_name
```

Comment:
Is this an `ERROR` or a `WARNING`? 

I recommend writing `Skipping the kprobe for {syscall} due to missing parameter list`.